api/models/Site.js

```diff
@@ -75,17 +75,7 @@ module.exports = {
   },
 
   afterCreate: function(model, done) {
-    Site.findOne({id: model.id }).populate('users')
-        .exec(function(err, site) {
-          if (err) return done(err);
-          if (!site.users[0]) return done();
-          var build = {
-              user: site.users[0].id,
-              site: model.id,
-              branch: model.defaultBranch
-          };
-          Build.create(build, done);
-        });
+    this.startInitialBuild(model, done)
```

Comment:
I made this change so I could stub the `startInitialBuild` callback instead of trying to stub `Site`'s `afterCreate` action. That is because of the way Waterline manages lifecycle callbacks which makes it very difficult to stub this callback directly.